.github/workflows/update-snippets.yml

```diff
@@ -0,0 +1,70 @@
+name: Update Snippets
+# Update cylc-snippets.json using bin/cylc-snippets.
+
+on:
+  # Ideally people making changes should re-run this task, but probably
+  # won't remember - I think once a month is probably often enough.
+  schedule:
+    - cron: "* * 15 * *"
+  workflow_dispatch:
+
+jobs:
+  snippets:
+    runs-on: ubuntu-latest
+    outputs:
+      create_pr: ${{ steps.set_output.outputs.create_pr }}
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install Cylc
+      run: pip install cylc-flow
+
+    - name: Update Snippets
+      run: ./bin/get_keywords > cylc-snippets.json
+
+    - name: Configure git
+      uses: cylc/release-actions/configure-git@v1
+
+    - name: git diff and set output
+      # Check whether any changes have been made to config.
+      id: set_output
+      run: |
+        if [[ -n $(git status cylc-snippets.json) ]]; then
+          echo "create_pr=true" >> $GITHUB_OUTPUT
+        else
+          echo "create_pr=false" >> $GITHUB_OUTPUT
+        fi
+
+  commit:
+    runs-on: ubuntu-latest
+    needs: [snippets]
+    if: needs.snippets.outputs.create_pr == 'true'
+    steps:
+
+    - uses: actions/checkout@v4
+
+    - name: Install Cylc
+      run: pip install cylc-flow
+
+    - name: Update Snippets
+      run: ./bin/get_keywords > cylc-snippets.json
+
+    - name: Configure git
+      uses: cylc/release-actions/configure-git@v1
```

Comment:
Duplication - why not just one job?

Reply:
I want the second job only to run if the first one finishes - probably needs to be an artifact storing workflow or something - any pointers?

Reply:
But you can do that in one job
```yml
- name: Checkout PR branch etc
  if: steps.set_output.output.create_pr == 'true'
  ...
```